packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextLayoutManager.java

```diff
@@ -146,13 +146,42 @@ public static boolean isRTL(MapBuffer attributedString) {
         == LayoutDirection.RTL;
   }
 
-  private static Layout.Alignment getTextAlignment(MapBuffer attributedString, Spannable spanned) {
+  @Nullable
+  private static String getTextAlignmentAttr(MapBuffer attributedString) {
```

Comment:
Is splitting the alignment methods related to core change, or just for consistency?

Reply:
It's just for consistency and to not retrieve alignment twice from `MapBuffer` (first from `getTextAlignment` and second from `getTextJustificationMode`)